### PR TITLE
Initialize the floorplan inside HTML via the JS api.

### PR DESCRIPTION
**Pull request by zergov**
## Based on #13 

This PR change the way the Floorplan is initialized inside a web page.

The old behavior was that the Floorplan initialization was hardcoded in the `index.html` page contained in the project.

This is not representative of the future usage of this application. Therefore, it has been change for the following expected behavior. 

The expected behavior is to create and initialize a `floorplan` object in a script tag inside the HTML container that is embedding the floorplan front-end application.

The floorplan object __must__ receive a configuration object containing the id of the div that is going to contain the Floorplan. The floorplan is then initialized when the `init()` function is called.

At this point, the floorplan is initialized and the rendering starts.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,34 +5,47 @@
...
+    </div>
   </body>
+  <script>
+
+    /**
+     *
+     *  Floorplan init goes here.
+     *
+     */
+    const floorplan = new floorplanets.Floorplan({
```
Comment (zergov) [inline public/index.html]: `floorplanets` c'est pas bin beau comme nom de closure, avez vous une idée de nom plus nice ?

Comment (ppot) [inline public/index.html]: `Flib` , `Fseats`, `Fets`, `F_ETS`

Comment (aviau) [inline public/index.html]: floorplanets c excellent.

Comment (Shuhala) [inline public/index.html]: floorplanets est le meilleur nom que tu pouvais trouver.

Comment (gperiard) [inline public/index.html]: :+1: pour floorplanets

Comment (zergov) [inline public/index.html]: :1st_place_medal:

**Pull request merged**